config/webpack.config.dev.js

```diff
@@ -25,7 +25,7 @@ const publicUrl = '';
 const env = getClientEnvironment(publicUrl);
 
 // List of all component names for generating the corresponding html pages.
-const components = ['button'];
+const components = ['button', 'fab'];
```

Comment:
new line!

otherwise we merge conflict constantly!

Reply:
Yeah, I thought that too... the problem is the damage has already been done with multiple pending PRs modifying these lines :(